be/src/vec/functions/comparison_equal_for_null.cpp

```diff
@@ -75,33 +79,62 @@ class FunctionEqForNull : public IFunction {
         } else if (left_only_null) {
             auto right_type_nullable = col_right.type->is_nullable();
             if (!right_type_nullable) {
+                // right_column is not nullable, so result is all false.
                 block.get_by_position(result).column =
                         ColumnVector<UInt8>::create(input_rows_count, 0);
```

Comment:
these result columns could be wrap in `ColumnConst`. if then we use this column, it could be accelerated

Reply:
Can not do this, return ColumnConst may have conflict with default impl for const.